src/table/hooks/__tests__/use-keyboard-active-listener.spec.tsx

```diff
@@ -33,8 +41,16 @@ describe("use-keyboard-active-listener", () => {
     jest.restoreAllMocks();
   });
 
+  /**
+   * since we are using useOnPropsChange
+   * the functions we expect to be called
+   * will not be triggered at first render
+   * that is why we use rerender() in below tests
+   */
   it("should call findCellWithTabStop and updateFocus with focusType blur when keyboard.active is false", () => {
-    renderUseKEyboardActiveListener();
+    const { rerender } = renderUseKeyboardActiveListener();
+    rerender();
```

Comment:
you could just do the rerender in `renderUseKeyboardActiveListener` instead of adding it in every test